cmd/snap/color.go

```diff
@@ -169,8 +171,11 @@ func longPublisher(esc *escapes, storeAccount *snap.StoreAccount) string {
 		return esc.dash + esc.green + esc.end
```

Comment:
do we know from design whether the star should be in green or some other color (yellow?) ?

Reply:
I talked to Holly and she advised to use "bright-yellow" from https://en.wikipedia.org/wiki/ANSI_escape_code#Colors as the color. I will adjust the code.